class_six/index.html

```diff
@@ -18,20 +18,22 @@ <h1>Grid Layout</h1>
         <div id="item1">Header</div>  
```

Comment:
This element is supposed to hold your logo change the text copy to logo or better if you have a logo place here